fe/fe-core/src/main/java/com/starrocks/sql/ast/QueryStatement.java

```diff
@@ -0,0 +1,27 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.RedirectStatus;
+import com.starrocks.analysis.StatementBase;
+
+public class QueryStatement extends StatementBase {
```

Comment:
Why not directly use `QueryRelation` like `TableFunctionRelation`.  sometimes, the parse result  is QueryStatement，sometimes ,the parse result is relation. which is confusing

Reply:
QueryRelation and QueryStatement are both ASTs, but they are subordinate relationships, and there is no mixed use scenario. The outermost ast of any sql is "statement", such as queryStatement, showStatement, alterStatement. And QueryRelation is just a "QueryBody" in queryStatement